Update docs (#169)
k-orc/openstack-resource-controller: website/docs/writing-controllers.md

```

> When the code compiles, enable the controller by starting it from `cmd/manager/main.go`.

Reviewer: ```suggestion
When the code compiles, enable the controller by adding it to `cmd/manager/main.go`.
```
